tests/unit/test_stats.py

```diff
@@ -298,3 +395,17 @@ def calc_correl(x, y):
         if file_string == "time_variance_rank_correlation":
             assert np.isclose(expect_var, test_results.iloc[1].iloc[1])
         os.remove(test_file)
+
+
+def test_quantile_values_above_1():
+    # We construct a scenario that can return quantile values above 1 if the numerical
+    # sums are not handled properly
+    to_quant = pd.Series(
+        [38751.41118, 40656.52654, 40656.52654, 40656.52654, 40656.52654, 42656.04102]
+    )
+    fringe = max(to_quant) - min(to_quant)
+    infiller = pd.Series(
+        np.arange(min(to_quant) - fringe, max(to_quant) + fringe, 1000)
```

Comment:
Do we need tests of what happens if we had say `np.arange(min(to_quant) - 10*fringe, max(to_quant) + 10*fringe, 1000)` or `np.arange(min(to_quant) - 0.1*fringe, max(to_quant) + 0.1*fringe, 1000)`?

I'm a bit unsure what is actually being tested here.

Reply:
We are testing that it returns the expected values both inside the range of input values and also outside it. The latter of these suggestions is almost a subset of the former, we can use the former instead of the current if you want but other than fishing for numerical issues I don't see what it does differently.

Reply:
I wasn't sure whether using `min-fringe` and `max+fringe` was a deliberate choice because it matches what is in the implementation. If a user passes values which are lower than `min+fringe`, is that a special case which we should test explicitly?

> The latter of these suggestions is almost a subset of the former

Ok makes sense

Reply:
I've increased the range and put in a zero floor test in case we do something crazy in the future